java_training/src/com/lftechnology/jan7/FileUtility.java

```diff
@@ -0,0 +1,159 @@
+package com.lftechnology.jan7;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class has methods to test a file, create directory, rename a file,delete a file and read a file
+ * 
+ * @author nerisa
+ * 
+ */
+public class FileUtility {
+	private static final Logger LOG = Logger.getLogger(FileUtility.class.getName());
+	private static Scanner s;
+	private static List<Integer> firstNum = new ArrayList<Integer>();
+	private static List<Float> secondNum = new ArrayList<Float>();
+	private static List<String> name = new ArrayList<String>();
+
+	/**
+	 * Creates a text file and checks whether that file is exists. Using the command exists(), isDirectory(), isFile(), getName() and
+	 * getAbsolutePath().
+	 * 
+	 * @param pathName
+	 *            Name of the file to be created
+	 */
+	public void testAFile(String pathName) {
+		Path file = Paths.get(pathName);
+		try {
+			Files.createFile(file);
+			LOG.log(Level.INFO, "File created:{0}", file.toAbsolutePath());
+		} catch (FileAlreadyExistsException e) {
+			LOG.log(Level.SEVERE, "{0} file already exists", file.getFileName());
+		} catch (IOException e) {
+			LOG.log(Level.SEVERE, "create file error. cause: {0}", e.getCause() + e.getMessage());
+		}
+		LOG.log(Level.INFO, "Directory?:{0}", Files.isDirectory(file));
```

Comment:
Also when we need to display other parameters it becomes easy to add them using LOG.log()